app/views/users/two_factor_authentication/setup/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1 class="govuk-heading-xl">Set up two-factor authentication</h1>
+
+<p class="govuk-body">
+  You need to set up two-factor authentication (2FA) on your account, for additional security.
+</p>
+
+<%= form_with method: :get do |form| %>
```

Comment:
this will generate an AJAX form by default, which causes my local branch to loop forever (i.e the page doesn't change, so the next template isn't rendered). You can pass `local: true` to disable that behaviour.

Reply:
Thanks. Weird that this used to work before a recent sync from main branch.

Reply:
As mentioned in the [Trello](https://trello.com/c/OeWE8SWL), this bug seems to stem from a recent change in the main branch.

Reply:
I've just rebased and pushed. Having tested this again, it seems to be working fine now without the `local: true` option, so I'm perplexed! Can you try and see if it's working for you locally as well?